src/coredump/coredump.c

```diff
@@ -1587,6 +1587,14 @@ static int process_kernel(int argc, char* argv[]) {
                 /* OK, now we know it's not the journal, hence we can make use of it now. */
                 log_set_target_and_open(LOG_TARGET_JOURNAL_OR_KMSG);
 
+        /* Log minimal metadata now, so it is not lost if the system is about to shut down. */
+        log_info("PID %s (%s) crashed with signal %s/%s",
+                 context.meta[META_ARGV_PID],
+                 context.meta[META_EXE] ? context.meta[META_EXE] : "unknown",
+                 context.meta[META_ARGV_SIGNAL],
+                 (safe_atoi(context.meta[META_ARGV_SIGNAL], &signo) >= 0 && SIGNAL_VALID(signo))
```

Comment:
signal_to_string() is happily handles signal numbers that are out of range, hence you can drop the extra SIGNAL_VALID() check here